content/content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -1154,6 +1155,9 @@ public static void publicDisplayChoicesIntoContext(SessionState state, Context c
 			}
 			try
 			{
+				if (!Validator.checkFilename(name)) {
+					throw new InvalidFilenameException(name);
```

Comment:
Same pattern

Reply:
Yeah, agree, the pattern is kinda bad, going to replace it.